libs/json-schema-to-zod-v3/src/__tests__/security.spec.ts

```
@@ -0,0 +1,244 @@
+      });
+
+      const config = getSecurityConfig();
+
+      expect(config.maxPatternLength).toBe(500);
+      expect(config.throwOnUnsafe).toBe(true);
+      expect(config.enableProtection).toBe(true); // unchanged
+    });
+
+    it('should not mutate returned config', () => {
+      const config1 = getSecurityConfig();
+      // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)